assets/pages/error/error500.jsx

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+import { statelessInit as pageInit } from 'helpers/page/page';
+import { renderPage } from 'helpers/render';
+
+import ErrorContent from './components/errorContent';
+
+
+// ----- Page Startup ----- //
+
+pageInit();
+
+
+// ----- Render ----- //
+
+const content = (
+  <ErrorContent
+    errorCode="500"
```

Comment:
Does this need to be less specific than just 500 codes? Not necessarily in this PR though because it already appears to include some refactoring of other things. A more generic 5xx (and 4xx) page might be helpful. I'm open to other opinions too though!